### ISSUE TITLE
Tests fails with yii2 master branch

### ISSUE THREAD

**Issue by turboezh** [yiisoft/yii2-redis#68]
### What steps will reproduce the problem?

Update yii2 in the test environment to the latest master commit or let `tests/bootstrap.php` download files from yii2 repo (`tests/ActiveRecordTestTrait.php`, `tests/BaseTestCase.php` and `tests/CacheTestCase.php`).
### What's expected?

All tests passes.
### What do you get instead?

```
SS.........................E.............E....................... 65 / 73 ( 89%)
........

Time: 7.55 seconds, Memory: 12.00Mb

There were 2 errors:

1) yiiunit\extensions\redis\ActiveRecordTest::testFindNestedRelation
yii\base\InvalidParamException: yiiunit\extensions\redis\data\ar\Customer has no relation named "ordersWithItems".

/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/BaseActiveRecord.php:1124
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/ActiveQueryTrait.php:196
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/ActiveQueryTrait.php:163
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/ActiveQuery.php:180
/home/alex/www/yii2/yii2-redis-dev/yii2/tests/framework/ar/ActiveRecordTestTrait.php:527
/home/alex/www/yii2/yii2-redis-dev/yii2/vendor/phpunit/phpunit/phpunit:36

Caused by
yii\base\UnknownMethodException: Calling unknown method: yiiunit\extensions\redis\data\ar\Customer::getordersWithItems()

/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/base/Component.php:285
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/BaseActiveRecord.php:1121
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/ActiveQueryTrait.php:196
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/vendor/yiisoft/yii2/db/ActiveQueryTrait.php:163
/home/alex/www/yii2/yii2-redis-dev/yii2-redis/ActiveQuery.php:180
/home/alex/www/yii2/yii2-redis-dev/yii2/tests/framework/ar/ActiveRecordTestTrait.php:527
/home/alex/www/yii2/yii2-redis-dev/yii2/vendor/phpunit/phpunit/phpunit:36

2) yiiunit\extensions\redis\ActiveRecordTest::testAfterRefresh
TypeError: Argument 1 passed to PHPUnit_Framework_TestCase::onNotSuccessfulTest() must be an instance of Exception, instance of Error given, called in /home/alex/www/yii2/yii2-redis-dev/yii2/vendor/phpunit/phpunit/src/Framework/TestCase.php on line 851

/home/alex/www/yii2/yii2-redis-dev/yii2/vendor/phpunit/phpunit/phpunit:36
```
### Additional info

| Q | A |
| --- | --- |
| Yii vesion | `2.0.7` in vendor/ and `master@edd6e71` in test env |
| PHP version | PHP 7.0.4-7ubuntu2 (cli) ( NTS ) |
| Operating system | Ubuntu 16.04 |

**Comment by cebe** [comment]
@turboezh yeah, this is a known issue. Error no 1 was introduced by @SilverFire and he wanted to fix that ;)
Error no 2 should be fixed with the release of yii 2.0.8.

**Comment by samdark** [comment]
@cebe are you sure about error no2? Issue title says "fails with yii2 master".

**Comment by turboezh** [comment]
Maybe I gave not enough details. As you know, there actually two yii2: in `vendor/` as dependency (I have 2.0.7 there) and as part of test env (`master@edd6e71` in this case), from which  ActiveRecordTestTrait.php, BaseTestCase.php and CacheTestCase.php are taken. If I set 2.0.7 in "test env" all tests passes.

Now I have told composer to use `*@dev` as dependency (with `master@edd6e71` in test env) and there only first error left.

I thinks, it's all ok. But in travis case it uses default composer dependency (2.0.7) and pull other needed files from master. This leads to fail all PRs.
